common/decklist.cpp

```diff
@@ -4,6 +4,8 @@
 #include <QCryptographicHash>
 #include <QDebug>
 #include "decklist.h"
+#include "../cockatrice/src/carddatabase.h"
```

Comment:
This can't work, since decklist.cpp is a common file used by both cockatrice and servatrice.
I recently had to face the same problem in this code to fix token loading (see #2670, commit 36f6907fa36b60bcc03e93ca08fd7b3e2da56533).
The idea is that since cockatrice's DeckLoader is a subclass of the common class DeckList, you can create a new virtual method in DeckList and reimplement it in DeckLoader in order to add the needed functionality.
In my example i added the getCardZoneFromName method to get the correct zone for a card from the db.
In your code, you could do the same to add a method that renames the card with its correct name.